docs/management/managing-saved-objects.asciidoc

```diff
@@ -1,81 +1,75 @@
 [[managing-saved-objects]]
-== Managing Saved Objects
+== Saved objects
 
-You can view, edit, delete, import, and export saved objects from 
-*Management > Saved Objects*. A saved object can be a search, visualization,
-dashboard, or index pattern.
-
-To narrow the list of saved objects, enter a search term or select an object 
-type.
+Saved objects store your visualizations, dashboards, maps, Canvas workpads, and more.
+You can view, edit, delete, import, and export saved objects in 
+*Management > Kibana > Saved Objects*.  
 
 [role="screenshot"]
-image::images/management-saved-objects.png[]
+image::images/management-saved-objects.png[Saved Objects]
 
 [float]
 [[managing-saved-objects-view]]
-=== View, edit, and delete objects
+=== View, edit, and delete
 
-Each object listed in *Saved Objects* has an *Actions* menu with two items:
+* To view and edit an object in its associated application, click the object title.
 
-* *In app* opens the object in its associated application so you can view and 
-edit it.
-* *Relationships* shows other objects that use this object so you know the 
-impact of deleting it.
+* To show objects that use this object, so you know the 
+impact of deleting it, click the actions icon image:images/actions_icon.png[Actions icon] 
+and select *Relationships*.
 
-To delete a saved object, select its checkbox and click *Delete*.
+* To delete one or more objects, select their checkboxes and click *Delete*.
```

Comment:
Since you complete one action after the other, shouldn't this be:

`...select their checkboxes, then click *Delete*.`